src/js/preloadWebComponents.js

```diff
@@ -23,6 +23,8 @@ module.exports = function() {
    if ((androidChrome || iosChrome) && needsWebComponents) {
       // This is requiring webcomponents.js@0.7.24 because that's what was used
       // by the Chromecast framework at the time this was added.
-      require('webcomponents.js'); // eslint-disable-line global-require
+      // We are using webcomponents-lite.js because it doesn't interfere with JQuery as
+      // badly.
```

Comment:
`as badly` is a little vague. Without of the context of this PR, the question that would likely be asked is "how was `webcomponents.js` interfering with jQuery?" Could we at least add a link to the original issue e.g. `as badly (see #21).`?

Reply:
It's just *bad* OK. 😸